include/gridtools/storage/info.hpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#pragma once
+
+#include <cassert>
+#include <cstddef>
+#include <type_traits>
+
+#include "../common/array.hpp"
+#include "../common/defs.hpp"
+#include "../common/generic_metafunctions/accumulate.hpp"
+#include "../common/host_device.hpp"
+#include "../common/layout_map.hpp"
+
+namespace gridtools {
+    namespace storage {
+        namespace info_impl_ {
+            template <class Layout>
+            GT_CONSTEXPR uint_t make_padded_length(Layout, int layout_arg, uint_t align, uint_t length) {
```

Comment:
Is there a reason that these functions are `GT_CONSTEXPR` and not `constexpr`?

Reply:
done